docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -293,6 +293,14 @@ repository. If you wish to search data across multiple regions, configure
 multiple clusters and use <<modules-cross-cluster-search,{ccs}>> or
 <<xpack-ccr,{ccr}>> instead of {search-snaps}.
 
+It's worth noting that if a searchable snapshot index has no replicas (as is the default
+in the cold tier), then when the node hosting it is shut down, allocation will immediately
```

Comment:
I'd prefer to avoid the default mention in this section, we speak of replicas just above this section.

```suggestion
It's worth noting that if a searchable snapshot index has no replicas, then when the node hosting it is shut down, allocation will immediately
```